.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Before submitting a pull request,** please make sure the following is done:
+
+1. Fork [the repository](https://github.com/fastify/fastify) and create your branch from `master`.
+2. Run `npm install` in the repository root.
+3. If you've fixed a bug or added code that should be tested, add tests!
+4. Ensure the test suite passes (`npm run test`).
+5. Run `npm run benchmark` to test the performance impact of your changes. Tip: `npm run bench` to compare branches interactively.
+6. If you haven't already, complete the CLA.
```

Comment:
I would prefer a checklist like node core does. It has less text to red through.

Reply:
Done